src/mixin-props-test.js

```diff
@@ -20,3 +20,61 @@ QUnit.test("basics", function(assert) {
 	el.age = "33";
 	assert.equal(el.age, 33, "updated age");
 });
+
+try {
+	eval(`
+		QUnit.test('Class fields should be observable', (assert) => {
+			class DefineElement extends mixinDefine(Object) {
+				greetings = 'Hello'; // jshint ignore:line
+				static get props() {
+					return {};
+				}
+			}
+			const el = new DefineElement();
+
+			if (el.hasOwnProperty('greetings')) {
+				el.initialize();
+			
+				el.on('greetings', () => {
+					assert.ok('The class field is observable');
+				});
+			
+				el.greetings = 'Hola';
```

Comment:
Can you also check that setting the value to a non-string works since expando properties like this are not typed.